examples/quickstart/registration-tpm.yaml

```diff
@@ -16,3 +16,6 @@ spec:
         debug: true
       registration:
         emulate-tpm: true
+  machineName: my-machine
```

Comment:
I think both of those were dropped due to user concerns, one becuase the location was confusing as an example and the other to not have a marked machienName and let the examples generate the machineName automatically